src/docs/03_system_scope_and_context.adoc

```diff
@@ -40,5 +40,17 @@ include::diagrams/03-02_TechnicalContext.puml[]
 | Users | People wanting to store and share their routes with control over their storage.
 | Viade | Our application that is supposed to be interoperable thanks to the use of Solid.
 | POD | Each user has a personal pod which stores all his information.
+| Maps API | Request are made to obtain the location and positions of routes and the geographical information is returned so the map can be rendered.
```

Comment:
I´d just put the second part about rendering the positions in the map, we don´t do any request to obtain any location/position.

Reply:
Fixed!